### PR TITLE
Update README.md to include expanded IDE compatibility matrix

### PR DESCRIPTION
**Pull request by parkjam4**
Starting a PR to update the readme file with the newly supported IDEs.
Waiting on the final list from the Eng team.

### CODE REVIEW

**Comment by etanshaul** [comment]
We should move the 2017.1 or later notice to apply to all of the IDEs, not just CE and UE.

**Comment by etanshaul** [comment]
@parkjam4 just fyi, when you merge in the PR, the commit title will be equal to the title of the commit in this PR. You can update it to be more descriptive (like the update I made to the PR title) before merging using GitHub's squash/merge UI (after clicking the button).

File: README.md [hunk 1]
```diff
@@ -29,14 +29,28 @@ For detailed user documentation go to our documentation
...
-In addition to IntelliJ IDEA, the plugin provides limited support for other JetBrains IDEs:
-
-| | IntelliJ IDEA - Ultimate | IntelliJ IDEA - Community | PyCharm |
+The Cloud Tools for IntelliJ plugin primarily supports the following IDEs, version 2017.1 or later:
+
+* IntelliJ IDEA Community Edition 
+* IntelliJ IDEA Ultimate Edition
+
+In addition to IntelliJ IDEA, the plugin provides limited support for the following IDEs, version 2017.1 or later:
+* PyCharm Professional Edition
```
Comment (etanshaul) [inline README.md]: just a single PyCharm entry is all we need. there is no difference in support between pro, community, and edu

Comment (parkjam4) [reply README.md]: How about PyCharm (all editions) to emphasize that you can use the plugins for all 3 editions?

Comment (etanshaul) [reply README.md]: can we just do PyCharm (Community and Professional)

to avoid taking up 2 lines?

**Pull request merged**